redirects.txt

```diff
@@ -1,4 +1,6 @@
-"porting/introduction.rst" "porting/Intro.rst"
+"porting/introduction.rst" "porting/index.rst"
+"porting/Intro.rst" "porting/index.rst"
+"porting/Index.rst" "porting/index.rst"
```

Comment:
this one we don't need now, right? I mean before this PR, there was never a mention of "porting/Index.rst", so we don't need to redirect it.

Reply:
without this it won't build for me locally

Reply:
strange. maybe after "make clean", like rm -rf _build or smt like that?

Reply:
hmmm with a squash rebase it does build fine without it, should i just force push?

Reply:
ok :+1:

Reply:
> strange. maybe after "make clean", like rm -rf _build or smt like that?

oh interesting it appears related to _build

Reply:
Yeah, I've seen smt similar. I guess the build process doesn't delete files from the build folder